content/configuration/index.md

```diff
@@ -414,7 +415,16 @@ module.exports = {
   },
 
   [node](node): {
-    /* TODO */
+    // Polyfills and mocks to run Node.js-
+    // environment code in non-Node environments.
+    
+    [console](node#node-console): false, // boolean | "mock"
+    [global](node#node-global): true, // boolean | "mock"
+    [process](node#node-process): true, // boolean
+    [__filename](node#node-__filename): "mock", // boolean | "mock"
+    [__dirname](node#node-__dirname): "mock", // boolean | "mock"
+    [Buffer](node#node-buffer): true, // boolean | "mock"
+    [setImmediate](node#node-): true // boolean | "mock" | "empty"
```

Comment:
@SterlingVix it looks like this hash link is failing:

``` bash
Fragment check: build/configuration/index.html --> node/#node-
null at build/configuration/index.html:387:32 <a class="code-link" href="node/#node-">...</a>
```

Can you double check to make sure you linked to the correct section here?